chrome_extension/mooltipass-content.js

```diff
@@ -1572,7 +1572,7 @@ cip.rememberCredentials = function(event, usernameField, usernameValue, password
 			});
 		}
 
-		var url = mpJQ(this)[0].action;
+		var url = mpJQ(event.target)[0].action;
```

Comment:
This line suppose to get action attribute of the form, but this evaluates to `cip` instance and not to DOM element. Changed it to `event.target` and now we are able to save injected proxy address. Take a look at this code in commit:

```js
// Set proxy address as action attribute of the form, so mcCombs will
// save credentials for the right url.
if (req.args[0].isProxy) {
  mpJQ('.mp-popup-http-auth__form').attr('action', req.args[0].proxyURL)
}
```